### PR TITLE
chore(docs): extend contributor docs by RBAC for stages

### PR DESCRIPTION
**Pull request by gu-stav**
### What does it do?

Extends contributor docs by RBAC for stages.

### Why is it needed?

Having complete docs is good.

### CODE REVIEW

**Comment by gu-stav** [comment]
@jhoward1994 I think the backend/ API docs also needs some information (especially the service docs). Do you want to add them to this PR or do you prefer another one?

**Comment by jhoward1994** [comment]
> @jhoward1994 I think the backend/ API docs also needs some information (especially the service docs). Do you want to add them to this PR or do you prefer another one?

Thanks I can add them here

**Comment by jhoward1994** [comment]
wdyt @gu-stav ? [`ac4bf84` (#18056)](https://github.com/strapi/strapi/pull/18056/commits/ac4bf84a6487bfb79c983c53e9eaf316546ced02)

There were details missing for assignees too, should I add them here or keep it focussed on RBAC?

**Comment by gu-stav** [comment]
> wdyt @gu-stav ? [`ac4bf84` (#18056)](https://github.com/strapi/strapi/pull/18056/commits/ac4bf84a6487bfb79c983c53e9eaf316546ced02)
> 
> There were details missing for assignees too, should I add them here or keep it focussed on RBAC?
> 
> 

Up to you. It would be nice to have complete docs and if you think you can split the work across the feature branch and main I'd split it - otherwise I think it's fine adding it here together 👍

**Comment by jhoward1994** [comment]
> > wdyt @gu-stav ? [`ac4bf84` (#18056)](https://github.com/strapi/strapi/pull/18056/commits/ac4bf84a6487bfb79c983c53e9eaf316546ced02)
> > There were details missing for assignees too, should I add them here or keep it focussed on RBAC?
> 
> Up to you. It would be nice to have complete docs and if you think you can split the work across the feature branch and main I'd split it - otherwise I think it's fine adding it here together 👍

Let's add it here if that's okay with you? Was there another part of the docs you thought needed work? That seems to be ever reference to review workflows

File: docs/docs/docs/01-core/admin/03-settings/01-review-workflows.mdx [hunk 1]
```diff
@@ -50,6 +50,12 @@ attribute will be re-ordered and not created.
 
 Stages without an `id` property will be created in the database on submission. Stages that existed already, but are not submitted again will be deleted on submission.
 
+When editing a workflow and `permissions` of a stage have not been modified by a user, they are sent as `undefined`. This special case allows users to edit a workflow without having to have
```
Comment (joshuaellis) [inline docs/docs/docs/01-core/admin/03-settings/01-review-workflows.mdx:53]:
```suggestion
When editing a workflow and the `permissions` of a stage have not been modified by a user, they are sent as `undefined`. This special case allows users to edit a workflow without having to have
```

**Review by joshuaellis — APPROVED** [review]
minor tweak but nice work folks ⭐

**Pull request merged**